nixos/modules/services/misc/paperless.nix

```
@@ -320,7 +321,7 @@ in
         # Enable internet access
         PrivateNetwork = false;
         # Restrict write access
-        BindPaths = [];
+        BindPaths = [ ];
```

Let's keep the diff minimal.
```suggestion
        BindPaths = [];
```